src/main/scala/com/socrata/geospace/lib/regioncache/MemoryManagingRegionCache.scala

```diff
@@ -45,6 +45,13 @@ abstract class MemoryManagingRegionCache[T](maxEntries: Int = 100, //scalastyle:
   protected def indicesBySizeDesc(): Seq[(RegionCacheKey, Int)]
 
   /**
+   * Returns keys ir order of least recently used to most used
+   * @return keys ir order of least recentlused to most used
+   */
+  protected def keysByLeastRecentlyUsed(): Seq[RegionCacheKey]
```

Comment:
I'd just define this as an `Iterator` instead of a `Seq`, instead of calling `toSeq` on `ascendingKeys` in the concrete implementation, and then calling `.iterator` on it again in `depressurizeByLeastRecentlyUsed`.